src/dsp_tools/utils/connection_live.py

```diff
@@ -51,18 +51,15 @@ def login(self, email: str, password: str) -> None:
             password: password of the user
 
         Raises:
-            BaseError: if DSP-API returns no token with the provided user credentials
+            PermanentConnectionError: if DSP-API returns no token with the provided user credentials
         """
         response = self.post(
             route="/v2/authentication",
             data={"email": email, "password": password},
         )
         if not response.get("token"):
-            raise BaseError(
-                f"Error when trying to login with user '{email}' and password '{password} "
-                f"on server '{self.server}'",
-                json_content_of_api_response=json.dumps(response),
-                api_route="/v2/authentication",
```

Comment:
The info of these 2 lines has already been logged on line 328 (`self._log_response(response)`)